apps/web/src/app/globals.css

```
@@ -10,40 +10,54 @@
+  --sidebar-accent: oklch(0.9251 0.0071 88.6450);
   --sidebar-accent-foreground: oklch(0.2417 0.0298 269.8827);
-  --sidebar-border: oklch(0.9251 0.0071 88.645);
-  --sidebar-ring: oklch(0.6511 0.0826 258.0981);
-  --font-sans: var(--font-sans), ui-sans-serif, sans-serif, system-ui;
-  --font-serif: var(--font-serif), Georgia, serif;
-  --font-mono: JetBrains Mono, Courier New, monospace;
-  --radius: 0.625rem;
+  --sidebar-border: oklch(0.9251 0.0071 88.6450);
+  --sidebar-ring: oklch(0.6333 0.0309 154.9039);
+  --font-sans: "Antic", ui-sans-serif, sans-serif, system-ui;
+  --font-serif: "Signifier", Georgia, serif;
```

Stylelint (17.4.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)